checks/evaluation/contributors.go

```diff
@@ -16,60 +16,38 @@ package evaluation
 
 import (
 	"fmt"
-	"sort"
-	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/contributorsFromOrgOrCompany"
 )
 
 const (
-	minContributionsPerUser    = 5
 	numberCompaniesForTopScore = 3
 )
 
-// Contributors applies the score policy for the Contributors check.
-func Contributors(name string, dl checker.DetailLogger,
-	r *checker.ContributorsData,
+func Contributors(name string,
+	findings []finding.Finding,
 ) checker.CheckResult {
-	if r == nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, "empty raw data")
-		return checker.CreateRuntimeErrorResult(name, e)
+	expectedProbes := []string{
+		contributorsFromOrgOrCompany.Probe,
 	}
 
-	entities := make(map[string]bool)
-
-	for _, user := range r.Users {
-		if user.NumContributions < minContributionsPerUser {
-			continue
-		}
-
-		for _, org := range user.Organizations {
-			entities[org.Login] = true
-		}
-
-		for _, comp := range user.Companies {
-			entities[comp] = true
-		}
-	}
-
-	names := []string{}
-	for c := range entities {
-		names = append(names, c)
+	if !finding.UniqueProbesEqual(findings, expectedProbes) {
+		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
+		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	sort.Strings(names)
+	reason := fmt.Sprintf("project has %d contributing companies or organizations", len(findings))
 
-	if len(names) > 0 {
-		dl.Info(&checker.LogMessage{
```

Comment:
can we use https://github.com/ossf/scorecard/blob/main/checker/check_result.go#L198 to log? (note: also changing in https://github.com/ossf/scorecard/pull/3409)

Reply:
Added but commented out because unit tests were failing.